content/en/docs/glossary/_index.md

```diff
@@ -75,6 +75,9 @@ Open Y
 Paragraph
 : A Drupal data structure that allows content editors to create sections of content on a page using preset groups of fields. See [Paragraphs on Drupal.org](https://www.drupal.org/project/paragraphs) and [Paragraphs in the distribution](/docs/user-documentation/paragraphs/).
 
+PEF
+: The "[Program Event Framework](/docs/development/program-event-framework/)", also known as PEF, consists of a group of [content types](/docs/user-documentation/content-types/activity-class-session/) and [front-end displays](/docs/user-documentation/schedules/) that can be used to create related programs, activities, classes, and sessions.
```

Comment:
![SCR-20240705-mlac](https://github.com/YCloudYUSA/yusaopeny_docs/assets/238201/1d0c59b1-dcf7-4713-aeaf-b97512f576b3)

In the [preview](https://deploy-preview-89--sparkling-souffle-4f78b0.netlify.app/docs/glossary/) something about the link processor is adding an extra space before the link, so just removing those quotes. 

```suggestion
: The [Program Event Framework](/docs/development/program-event-framework/), also known as PEF, consists of a group of [content types](/docs/user-documentation/content-types/activity-class-session/) and [front-end displays](/docs/user-documentation/schedules/) that can be used to create related programs, activities, classes, and sessions.
```